config/samples/nnf_v1alpha1_nnfcontainerprofile.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfContainerProfile
+metadata:
+  name: sample-nnfcontainerprofile
+  namespace: nnf-system
+data:
+  # Specifies the number of times a container will be retried upon a failure. A
+  # new pod is deployed on each retry. Defaults to 6 by kubernetes itself and
+  # must be set. A value of 0 disables retries.
+  retryLimit: 6
+
+  # Stop any containers after X seconds once a workflow has transitioned to
+  # PostRun. A value of 0 disables this behavior.
+  postRunTimeoutSeconds: 0
```

Comment:
Does this have a default?

Reply:
It essentially defaults to 0 with omitempty since it's an int.

Reply:
If you are using this sample as documentation, please include the default values

Reply:
Done